src/main/java/smartthings/cassandra/CassandraConnection.java

```diff
@@ -53,38 +62,44 @@ public CassandraConnection(MigrationParameters parameters, String ownerName) {
 		if (session == null) {
 			this.host = parameters.getHost();
 			this.port = parameters.getPort();
+			this.localDatacenter = parameters.getLocalDatacenter();
 			this.username = parameters.getUsername();
 			this.password = parameters.getPassword();
 			this.truststorePassword = parameters.getTruststorePassword();
 			this.truststorePath = parameters.getTruststorePath();
 			this.keystorePassword = parameters.getKeystorePassword();
 			this.keystorePath = parameters.getKeystorePath();
+			this.mySession = true;
 		}
 		this.keyspace = parameters.getKeyspace();
-
 	}
 
 	public void connect() throws Exception {
 		if (session == null) {
 			logger.debug("Connecting to Cassandra at " + host + ":" + port);
 
-			QueryOptions queryOptions = new QueryOptions().setConsistencyLevel(ConsistencyLevel.QUORUM);
-
-			Cluster.Builder builder = Cluster.builder().addContactPoint(host).withPort(port).withMaxSchemaAgreementWaitSeconds(20).withQueryOptions(queryOptions);
+			CqlSessionBuilder builder = CqlSession.builder()
+				.addContactPoint(new InetSocketAddress(host, port))
+				.withLocalDatacenter(localDatacenter)
```

Comment:
Cassandra Driver 4.3 will fail to startup when a host / port is specified without also setting the local datacenter field.  Added it as a settable parameter w/ a default of "datacenter1" which matches the drivers default value.

This has more details:
https://docs.datastax.com/en/developer/java-driver/4.2/manual/core/load_balancing/